packages/mwp-api-proxy-plugin/src/util/receive.js

```diff
@@ -260,7 +260,9 @@ export const makeInjectResponseCookies = request => ([response, _, jar]) => {
 	});
 };
 
-export const makeReceive = request => {
+// curried function(request, query, response) that will format an API response into
+// a 'standard' response format
+export const makeReceiver = request => {
```

Comment:
https://github.com/meetup/meetup-web-platform/pull/599/files#diff-5a9c646ca82547978ac644b96949f5c5R272 - sounds like a security issue, anything we could do about this?

Reply:
not really. I should actually move/remove that comment because it's the log function (`makeLogResponse`) that determines how much of an API response gets logged. Currently, we're actually not logging this info at all.

Reply:
oops ^ that was my comment :)